### PR TITLE
css-loader, uniforms added

### PR DESCRIPTION
**Pull request by TinaBryan**
Hello @zeroasterisk 

In the template/package.json

* added the npm css-loader 
* added uniforms-bootstrap4
* moved the uniforms dependencies to peerDependencies

### CODE REVIEW

File: packages/template/package.json [hunk 1]
```diff
@@ -18,41 +18,47 @@
...
   "peerDependencies": {
     "react": "^0.14.7 || ^15.0.0"
   },
   "dependencies": {
-    "babel-runtime": "^6.11.6"
+    "babel-runtime": "^6.11.6",
+    "react": "^15.3.2",
+    "react-dom": "^15.3.2",
+    "uniforms": "^1.5.0-rc.1",
+    "uniforms-bootstrap4": "^1.5.0-rc.1"
```
Comment (zeroasterisk) [inline packages/template/package.json]: lets move everything but `babel-runtime` to `peerDependencies`

We don't want to re-install a bunch of junk people already have in their project, nor force our versions, which may get out of date...  Lets do the broad-scope versions like `react` `"^0.14.7 || ^15.0.0"` meaning allow `0.14.*` or `15.*`.

File: packages/package.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (zeroasterisk) [inline packages/package.json]: I don't understand the purpose of this file

**Review by zeroasterisk — DISMISSED** [review]

File: packages/package.json (hunk 2 above)
Comment (TinaBryan) [reply packages/package.json]: Yes, you are right.  Deleting it.
Thank you :)

**Comment by zeroasterisk** [comment]
thanks.

I may not have been super clear, but the goal here, is to end up with a `template` package which renders in `storybook` with a basic `uniforms` form on it... No fancy inputs or anything yet, just trying to get the basics in place on the template, so we can replicate and build from there.

I saw in slack you were mentioning being unsure where the meteor stuff goes... there will be no meteor stuff at all in this repo.  We are trying to help make `uniforms` distance itself from meteor.

But uniforms wont work without a schema - and there is currently only SimpleSchema2 which has a bridge which doesn't need meteor.  (never used it yet, but it should work)

Also note: they are working on a GraphQLBridge implementation, which would also be really neat to use: https://github.com/vazco/uniforms/issues/118

Either of those should give us a (bridged) schema, and then let uniforms work.

File: packages/template/package.json (hunk 1 above)
Comment (TinaBryan) [reply packages/template/package.json]: @zeroasterisk 
- changes: moved everything except `bable-runtime` to `peerDependencies`

**Comment by TinaBryan** [comment]
@zeroasterisk 
- removed meteor from project
- reinstalled uniforms, uniforms-bootstrap4, uniforms-unstyled via npm
- Looking for source SimpleSchema2 that does not require meteor.  
  Is this the [SimpleSchema2Bridge](https://github.com/aldeed/node-simple-schema)
  we are looking for?

**Comment by zeroasterisk** [comment]
simple schema 2 is a branch, not yet fully released:
https://github.com/aldeed/meteor-simple-schema/tree/v2

Looks like it's `npm install simpl-schema`

**Review by errinjohnson — APPROVED** [review]

**Review by zeroasterisk — APPROVED** [review]

**Pull request merged**